docker/dev/docker-compose-gcp.yml

```diff
@@ -0,0 +1,194 @@
+version: '3.5'
+
+services:
+  postgres:
+    container_name: postgres
+    image: 'postgres:15'
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
+    ports:
+      - '5433:5432'
+    restart: unless-stopped
+    networks:
+      - testnet
+    volumes:
+      - pg-data:/var/lib/postgresql/data
+      - ./dbinit.sql:/docker-entrypoint-initdb.d/init.sql
+
+  wallet-frontend:
+    container_name: wallet-frontend
+    build:
+      context: ../..
+      dockerfile: ./packages/wallet-frontend/Dockerfile-GCP.dev
+    environment:
+      NODE_ENV: development
+      PORT: 4003
+      NEXT_PUBLIC_USE_TEST_KYC_DATA: 'true'
+      NEXT_PUBLIC_BACKEND_URL: http://35.196.11.156:3003
+      NEXT_PUBLIC_OPEN_PAYMENTS_HOST: http://rafiki-backend
+    restart: always
+    networks:
+      - testnet
+    ports:
+      - '4003:4003'
+
+  # Testnet
+  wallet-backend:
+    container_name: wallet-backend
+    build:
+      context: ../..
+      dockerfile: ./packages/wallet-backend/Dockerfile.dev
+    depends_on:
+      - postgres
+      - rafiki-backend
+    environment:
+      NODE_ENV: development
+      PORT: 3003
+      DB_URL: postgres://wallet_backend:wallet_backend@postgres/wallet_backend
+      JWT_ACCESS_TOKEN_SECRET: u3uakiThYILFEewwhlHkepwMLM@Ikbj1LA!JQeeVyVEn$$Cj9t$$Ix0TWMYrEnpq$$X
```

Comment:
I suggest to add the jwt tokens as it is done for rapyd

Reply:
I don't understand :)

Reply:
add them in .env file and here change it to `JWT_ACCESS_TOKEN_SECRET: ${JWT_ACCESS_TOKEN_SECRET}`

Reply:
I think this can be solved after it's updated in main docker file? not docker file for gcp.